### ISSUE TITLE
Accessing wrk command-line parameters from Lua script

### ISSUE THREAD

**Issue by defanator** [wg/wrk#171]
I'm wondering if there is a way to get some particular parameters like number of threads and connections - from the Lua script. I need to obtain number of threads on setup() stage to split a set of predefined requests across threads.

Probably it would be useful to let scripts access the whole "cfg" struct somehow.

**Comment by defanator** [comment]
Just for the record - I made some PoC on this here: https://github.com/defanator/wrk/tree/script_config

Any feedback greatly appreciated.

**Comment by mkarlovich** [comment]
@defanator Did you ever got this working "splitting a set of predefined requests across threads" in your fork?  I would like to try it out if you did and have an example.

**Comment by defanator** [comment]
@mkarlovich yes I did. In fact, you won't need any fork in order to achieve this, master branch of wrk can do it as well. I've been using this approach for running load test of external API service we were developing. There were 2 separate scripts iirc:
 - first one was used for mass registration of "fake" accounts with saving all the account information to a file,
 - second one was used for random load requests to an API endpoint using the credentials from the file delivered by the 1st script.

**Comment by mkarlovich** [comment]
I found a way opening a process using io.popen like was done here: http://www.puremistake.com/3/
Did you find a way without doing something like this?
